test/aliases.js

```diff
@@ -0,0 +1,59 @@
+import { join } from 'path'
+import { version } from 'process'
+
+import test from 'ava'
+
+import { resolveVersionRangeAlias } from '../src/aliases.js'
+
+const resolveInFolder = (versionRange, folder) =>
+  resolveVersionRangeAlias(versionRange, {
+    cwd: join(__dirname, 'fixtures', folder),
+  })
+
+test('Resolve - with node-version pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'node-version-project')
+  t.is(versionRange, 'v12.12', 'nvmrc version not resolved')
+})
+
+test('Resolve - with nvmrc pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'nvmrc-project')
+  t.is(versionRange, 'v12.14', 'nvmrc version not resolved')
+})
+
+test('Resolve - with nave pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'nave-project')
+  t.is(versionRange, 'v12.16', '.nave version not resolved')
+})
+
+test('Resolve - in mixed project pseudo version, .node-version having precedence over .nvmrc', async (t) => {
+  const versionRange = await resolveInFolder('.', 'mixed-project')
+  t.is(versionRange, 'v12.12', 'not resolved to .node-version')
+})
+
+test('Throw error if resolving - and none of the node version files are found', async (t) => {
+  await t.throwsAsync(
+    resolveVersionRangeAlias('.', { cwd: join(__dirname, '..', 'src') }),
```

Comment:
```suggestion
    resolveVersionRangeAlias('.', { cwd: `${__dirname}/../src` }),
```

Should work because `fs.*` core methods (which `find-up` relies on) work with forward slashes even on Windows.

Reply:
wasnt aware, will apply then